apps/ui/src/components/ProposalStatus.vue

```diff
@@ -36,6 +37,10 @@ defineProps<{ state: ProposalState }>();
       v-else-if="state === 'passed'"
       class="text-skin-bg inline-block size-[17px] mb-[1px]"
     />
+    <IH-minus-circle
```

Comment:
Can we use the solid version here?
```suggestion
    <IS-minus-circle
```